manifests/m/Microsoft/PIX/2303.30/Microsoft.PIX.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=AUSU.CRLF.5-1-26085-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: "2303.30"
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Microsoft
+PackageName: Microsoft PIX
+PackageUrl: https://devblogs.microsoft.com/pix/download/
+License: Proprietary
+# LicenseUrl:
+Copyright: Copyright (c) Microsoft Corporation. All rights reserved.
+# CopyrightUrl:
+ShortDescription: Performance tuning and debugging for DirectX 12 games on Windows
+# Description:
+Moniker: pix
+Tags:
+- d3d12
+- dxr
+Agreements:
+- AgreementLabel: End User License Agreement (EULA)
+  AgreementUrl: https://www.microsoft.com/en-us/research/wp-content/uploads/2017/01/Microsoft_Pix_EULA_20160621.pdf
```

Comment:
This isn't the right EULA! (see comment on https://github.com/microsoft/winget-pkgs/pull/146955)